packages/cli/internal/pkg/environment/environment.go

```diff
@@ -6,13 +6,13 @@ import (
 	"github.com/aws/amazon-genomics-cli/internal/pkg/aws/ecr"
 )
 
-const DefaultEcrRegistry = "555741984805"
+const DefaultEcrRegistry = "680431765560"
 const DefaultEcrRegion = "us-east-1"
 
 // TODO: Implement better tag versioning system
-const DefaultCromwellTag = "2021-10-01T21-33-26Z"
-const DefaultNextflowTag = "2021-10-01T21-33-26Z"
-const DefaultWesTag = "2021-10-01T21-33-26Z"
+const DefaultCromwellTag = "64"
+const DefaultNextflowTag = "21.04.3"
+const DefaultWesTag = "0.1.0"
```

Comment:
Will these need to be updated manually in the future?

Reply:
Yes. You can think of them as dependencies that AGC relies on. If we want to use a newer Cromwell version, we update the tag here.

Ultimately I'd like these to live in some sort of "package.json" file for AGC, but for now we're just hardcoding it here.